lib/config.js

```diff
@@ -88,6 +88,12 @@ module.exports = class Config {
       res.server.middleware.unshift(hygienist(res.server.server.baseDir))
     }
 
+    // webpack must consume an array for the value in our entry object
+    for (let key in res.entry) {
+      let value = res.entry[key]
+      if (typeof value === 'string') res.entry[key] = Array(value)
+    }
```

Comment:
So you can do `res.entry[key] = Array.prototype.concat(res.entry[key])` here to one-line it 😁